docs/pages/versions/unversioned/sdk/system-ui.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SystemUI
+sourceCodeUrl: 'https://github.com/expo/expo/tree/master/packages/expo-system-ui'
+---
+
+import APISection from '~/components/plugins/APISection';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight'
+import InstallSection from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+
+**`expo-system-ui`** provides access to the underlying System UI features such as changing the Android Navigation Bar's color or visibility, adjusting the app's theme, making the iOS Home Indicator sticky, and more.
+
+<PlatformsSection android emulator ios simulator />
+
+## Installation
+
+<InstallSection packageName="expo-system-ui" />
+
+## API
+
+```js
+import { SystemUI } from 'expo-system-ui';
```

Comment:
```suggestion
import * as SystemUI from 'expo-system-ui';
```

Reply:
modify the JS API to ensure this type of import is correct.

Reply:
Are you sure? I kinda thought it's weird to be able to use the API like this:

```js
import { setNavigationBarColor } from 'expo-system-ui'
```

Because as a developer it's simpler and more nicely structured to write
```js
SystemUI.
```

...and then press "auto-import" instead of using dangling, loosely-floating-around functions like

```js
setNavigationBarColor(
```

and then pressing import.

Reply:
Also the `import * as SystemUI` syntax is (from my experience) waaaaay less common, because you use named exports and let the IDE/Editor auto-import for you.